client/rest-high-level/src/main/java/org/elasticsearch/client/IndicesClient.java

```diff
@@ -836,4 +837,50 @@ public void analyzeAsync(AnalyzeRequest request, RequestOptions options,
         restHighLevelClient.performRequestAsyncAndParseEntity(request, IndicesRequestConverters::analyze, options,
             AnalyzeResponse::fromXContent, listener, emptySet());
     }
+
```

Comment:
pls group the sync/async together the same way we do it for the other methods in the client.